department-of-veterans-affairs/vets-website
# VAGOV-3663: Updating graphql query syntax to match updated module schema

## Description
This PR contains query changes required for compatibility with the latest version of the Drupal graphQL module. 

**These updates require the following PR: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/323 to be merged into va.gov-cms or the front-end build will break.** 

Summary of graphQL query changes:
-  Underscores only in the beginning of image style names (i.e CROP_7_2 changes to CROP72, _1_1_SQUARE_MEDIUM_THUMBNAIL changes to _11SQUAREMEDIUMTHUMBNAIL)
-  entity reference field queries have to reference the appropriate node type. (i.e. FieldNodePageFieldSupportServices changes to FieldNodeLandingPageFieldSupportServices in landingPage.graphql.js)

For more background see conversation in this JIRA ticket: https://va-gov.atlassian.net/browse/VAGOV-3026

## Testing done
To Test:
- Pull down this vagov.cms PR and build your Lando instance with it: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/323
- Pull down this vets-website PR and build it using content pulled from your running Lando instance: `npm run watch:static -- --pull-drupal --drupal-address=http://va-gov-cms.lndo.site`
- Confirm front-end build completes successfully with no errors
- Navigate to `http://localhost:3001/drupal/debug/` pick some pages at random and make sure all content is loaded correctly.
- To make extra sure everything is working, go to `http://va-gov-cms.lndo.site/admin/reports/dblog` and make sure there are no graphQL related errors in the log. 

## Screenshots


## Acceptance criteria

## Definition of done
